homeassistant/components/smarttub/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""SmartTub integration."""
+import logging
+
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN, SMARTTUB_CONTROLLER, UNSUB_UPDATE_LISTENER
+from .controller import SmartTubController
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["climate"]
+
+
+async def async_setup(hass, _config):
+    """Set up smarttub component."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    return True
+
+
+async def async_setup_entry(hass, entry):
+    """Set up a smarttub config entry."""
+
+    controller = SmartTubController(hass)
+    hass.data[DOMAIN][entry.unique_id] = {
+        SMARTTUB_CONTROLLER: controller,
+    }
```

Comment:
I think you can move this below and combine with lines 38-40

Reply:
Good point, done